src/metal/metal-device.cpp

```
@@ -543,6 +543,23 @@ void DeviceImpl::registerResource(MTL::Resource* resource)
     }
 }
 
+bool DeviceImpl::registerExternalResource(MTL::Resource* resource)
```

Cppcheck (2.20.0)

[style] The function 'registerExternalResource' is never used.

(unusedFunction)